### PR TITLE
updated readme for sqrt documentation

### PR DESCRIPTION
**Pull request by leesafini**
AnalyticalGraphicsInc/cesium/pull/4606

- Added documentation for `sqrt`
- Added self to Acknowledgments

### CODE REVIEW

File: Styling/README.md [hunk 1]
```diff
@@ -501,6 +502,18 @@ Returns the cosine value of the number (in radians).
 }
 ```
 
+#### sqrt 
+
+`sqrt(value : Number) : Number`
```
Comment (pjcozzi) [inline Styling/README.md]: What is the range of `value`?  `>= 0.0`?  What is the behavior if, for example, `-1.0` is provided?

File: Styling/README.md [hunk 2]
```diff
@@ -501,6 +502,18 @@ Returns the cosine value of the number (in radians).
...
+
+Returns the square root value of the number.
+
+```json
+{
+    "show" : "sqrt(${temperature}) > 0.0"
```
Comment (pjcozzi) [inline Styling/README.md]: Perhaps use a more realistic example like use `sqrt` to create a color ramp using the `"color"` property instead of `"show"`.

**Comment by leesafini** [comment]
Made changes based on your comments @pjcozzi. Was this what you had in mind for the color ramp?

**Comment by pjcozzi** [comment]
> Was this what you had in mind for the color ramp?

Yes, but use `expression` so it is more efficient.  Also, only two or three conditions are needed to get the point across; it's a good practice to make the example as simple as possible, but no more simple.  Here's an example from the spec:

```
{
    "color" : {
        "expression" : "regExp('^1(\\d)').exec(${id})",
        "conditions" : [
            ["${expression} === '1'", "color('#FF0000')"],
            ["${expression} === '2'", "color('#00FF00')"],
            ["true", "color('#FFFFFF')"]
        ]
    }
}
```

> Also, should I be adding myself to the contributers list for this specific page?

Let's stick with the main acknowledgements for now; we'll likely update each page to have editors and contributors like the [glTF spec](https://github.com/KhronosGroup/glTF/blob/master/specification/README.md) and then we would move your name there; right now "contributors" kind of means "editors."

**Comment by leesafini** [comment]
@lilleyse updated!

File: Styling/README.md [hunk 3]
```diff
@@ -501,6 +502,25 @@ Returns the cosine value of the number (in radians).
...
+Returns the square root of value if value >= 0. Returns NaN when value < 0.
+
+```json
+{
+    "color" : {
+    "expression" : "sqrt(${temperature}",
+    "conditions" : [
+        ["${expression} >= 0.5", "color('#00FFFF')"],
+        ["${expression} >= 0.0", "color('#FF00FF')"],
+        ["true", "color('#FFFFFF')"]
```
Comment (lilleyse) [inline Styling/README.md]: You can remove this last line since it should always meet the two previous conditions.

Comment (leesafini) [inline Styling/README.md]: done

File: Styling/README.md [hunk 4]
```diff
@@ -501,6 +502,24 @@ Returns the cosine value of the number (in radians).
...
+
+```json
+{
+    "color" : {
+    "expression" : "sqrt(${temperature}",
+    "conditions" : [
+        ["${expression} >= 0.5", "color('#00FFFF')"],
+        ["${expression} >= 0.0", "color('#FF00FF')"]
+    ]
+    }
```
Comment (lilleyse) [inline Styling/README.md]: The indentation is still a bit off.

Comment (leesafini) [inline Styling/README.md]: What is it supposed to be?

Comment (lilleyse) [inline Styling/README.md]:
```
"color" : {
    "expression" : "sqrt(${temperature}",
    "conditions" : [
        ["${expression} >= 0.5", "color('#00FFFF')"],
        ["${expression} >= 0.0", "color('#FF00FF')"]
    ]
}
```

**Comment by lilleyse** [comment]
Thanks @leesafini!

**Pull request merged**